packages/auth/android/src/main/java/io/invertase/firebase/auth/ReactNativeFirebaseAuthModule.java

```diff
@@ -1950,6 +2074,7 @@ private WritableMap getJSError(Exception exception) {
     String message = exception.getMessage();
     String invalidEmail = "The email address is badly formatted.";
 
+    System.out.print(exception);
```

Comment:
Dumping an error to System.out is discouraged, if it is already being propagated up, then I think this could be removed?